### PR TITLE
Update documentation for 0.9.7

### LINKED ISSUE #608: Add "build to tar" task/mojo documentation

### LINKED ISSUE #644: Add "useCurrentTimestamp" to READMEs

### PR DESCRIPTION
**Pull request by TadCordle**
To be merged after 0.9.7 is released. Fixes #608, fixes #644.

Other stuff we need to add:
- [x] ~~`sendCredentialsOverHttp` system property~~ Since this is insecure, it's probably best undocumented and left for people to find after seeing the initial error.
- [x] `jib.httpTimeout` system property

### CODE REVIEW

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -149,7 +163,7 @@ Field | Type | Default | Description
 `to` | [`to`](#to-closure) | *Required* | Configures the target image to build your application to.
 `container` | [`container`](#container-closure) | See [`container`](#container-closure) | Configures the container that is run from your built image.
 `useProjectOnlyCache` | `boolean` | `false` | If set to true, Jib does not share a cache between different Maven projects.
-`allowInsecureRegistries` | boolean | `false` | If set to true, Jib uses HTTP as a fallback for registries that do not support HTTPS. Leaving this parameter set to false is strongly recommended, since communication with insecure registries is unencrypted and visible to others on the network.
+`allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib uses HTTP as a fallback for registries that do not support HTTPS. Leaving this parameter set to false is strongly recommended, since communication with insecure registries is unencrypted and visible to others on the network.
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md]: This is misleading, so I'd like to change it to "... that do not support HTTPS or whose certificates cannot be verified" or something like that until we fix #643.

File: jib-maven-plugin/README.md [hunk 2]
```diff
@@ -213,8 +227,9 @@ Property | Type | Default | Description
 `jvmFlags` | list | *None* | Additional flags to pass into the JVM when running your application.
 `mainClass` | string | *Inferred\** | The main class to launch the application from.
 `args` | list | *None* | Default main method arguments to run your application with.
-`ports` | `List<String>` | *None* | Ports that the container exposes at runtime (similar to Docker's [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction).
+`ports` | list | *None* | Ports that the container exposes at runtime (similar to Docker's [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction).
 `format` | string | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
+`useCurrentTimestamp` | boolean | `false` | By default, Jib wipes all timestamps to guarantee reproducibility. If this parameter is set to true, Jib will set the image's creation timestamp to the time of the build, which sacrifices reproducibility for easily being able to tell when your image was created.
```
[the remark this answers is not in the document]
Comment (chanseokoh) [reply jib-maven-plugin/README.md]: boolean --> \`boolean\` for consistency?

[the remark this answers is not in the document]
Comment (TadCordle) [reply jib-maven-plugin/README.md]: For some reason the primitive types don't use code formatting in the maven readme, but they do in the gradle readme, so I left it the way it is to be consistent within this file. But yeah I don't think it'd hurt to change true to `true`.

File: jib-maven-plugin/README.md [hunk 3]
```diff
@@ -213,8 +227,15 @@ Property | Type | Default | Description
...
 `mainClass` | string | *Inferred\** | The main class to launch the application from.
 `args` | list | *None* | Default main method arguments to run your application with.
-`ports` | `List<String>` | *None* | Ports that the container exposes at runtime (similar to Docker's [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction).
+`ports` | list | *None* | Ports that the container exposes at runtime (similar to Docker's [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction).
 `format` | string | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
+`useCurrentTimestamp` | boolean | `false` | By default, Jib wipes all timestamps to guarantee reproducibility. If this parameter is set to `true`, Jib will set the image's creation timestamp to the time of the build, which sacrifices reproducibility for easily being able to tell when your image was created.
+
+You can also configure HTTP connection/read timeouts for registry interactions using the `jib.httpTimeout` system property, configured in milliseconds via commandline (you can also set it to `0` for infinite timeout):
```
Comment (chanseokoh) [inline jib-maven-plugin/README.md]: Do you think it's worth mentioning the default is 20000 (20 seconds)?

File: jib-gradle-plugin/README.md [hunk 4]
```diff
@@ -183,6 +197,13 @@ Property | Type | Default | Description
 `args` | `List<String>` | *None* | Default main method arguments to run your application with.
 `ports` | `List<String>` | *None* | Ports that the container exposes at runtime (similar to Docker's [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction).
 `format` | `String` | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
+`useCurrentTimestamp` | `boolean` | `false` | By default, Jib wipes all timestamps to guarantee reproducibility. If this parameter is set to `true`, Jib will set the image's creation timestamp to the time of the build, which sacrifices reproducibility for easily being able to tell when your image was created.
+
+You can also configure HTTP connection/read timeouts for registry interactions using the `jib.httpTimeout` system property, configured in milliseconds via commandline (you can also set it to `0` for infinite timeout):
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md]: Do you think it's worth mentioning the default is 20000 (20 seconds)?

File: jib-maven-plugin/README.md (hunk 3 above)
Comment (TadCordle) [reply jib-maven-plugin/README.md]: Hmm yeah I think that'd be helpful.

**Comment by coollog** [comment]
Let's add the CHANGELOG update here too.

**Comment by coollog** [comment]
Let's add a CHANGELOG entry for the Maven settings decryption fix.

**Pull request merged**